src/holofuel/pages/Inbox/Inbox.js

```diff
@@ -279,7 +285,7 @@ export function TransactionRow ({ transaction, setConfirmationModalProperties, i
   const [highlightGreen, setHighlightGreen] = useState(false)
   const [highlightRed, setHighlightRed] = useState(false)
   const [isDisabled, setIsDisabled] = useState(false)
-  const [isLoading, setIsLoading] = useState(false)
+  const [isLoading, setIsLoadingRaw] = useState(false)
```

Comment:
This is very particular, but can we update the const name to match the change in the function name?  IMO, it helps idenfity the pairs well.

Reply:
the issue is `isLoading` is good name for the state. That's what the state represents. I've used this convention in other projects when I wanted the setter function to do more than just set the state. Two possibilities:

1) we adopt this convention (the convention is `set[VariableName]Raw` is an acceptable name for a state setter, in addition to `set[VariableName]`, so whenever we see a setter that ends in `Raw` we know to just ignore that).

2) I rename the setter here back to `setIsLoading` and change the `setIsLoading` function later in this function to be called something else. `setIsLoadingAndAreActionsPaused` makes sense but is a mouthful. 

I have a slight preference for 1), but I'm used to it so if you feel strongly that you don't want to do that I'm happy to do 2)

Reply:
Got it.  I like the vibe of #2, but agree that the name is a bit long.  This function's purpose is to pause the actionable slider for all other transactions on the page, so how about renaming the function to `SetIsPaused` or `PausePageActions`?  Thoughts?

Reply:
well, the function is both to set paused and set loading. Set paused is dependent on set loading, so thats why I went with setLoading originally.

But I'll figure something out

Reply:
Ok changed it to `setIsLoadingAndPaused`

How's that?